conbench/api/index.py

```diff
@@ -85,20 +85,28 @@ def get(self):
           - Ping
         """
         now = datetime.datetime.now(datetime.timezone.utc)
-        query = text("SELECT version_num FROM alembic_version")
-        try:
-            # In local deployment the PostgreSQL server logs: `ERROR:  relation
-            # "alembic_version" does not exist at character 25`. That is
-            # probably because not schema migration has yet been performed?
-            # Should we make it so that this table exists also for those
-            # dev/from-scratch deployments?
-            version = list(Session.execute(query))[0]["version_num"]
-        except:
-            version = "unknown"
+
+        if Config.TESTING:
+            alembic_version = "not-set-in-testing"
+        else:
+
+            query = text("SELECT version_num FROM alembic_version")
+            try:
+                # In local deployment the PostgreSQL server logs: `ERROR:  relation
+                # "alembic_version" does not exist at character 25`. That is
+                # probably because not schema migration has yet been performed?
```

Comment:
Typo?

```suggestion
                # probably because no schema migration has yet been performed?
```

Reply:
Also I know this isn't your text — just moving, but it stood out anyawy!

Reply:
Yes! Will change that in another branch.